notes/stv/20210128-Spark-tmp-space-deploy.txt

```diff
@@ -0,0 +1,138 @@
+        --volume "${AGLAIS_CODE:?}/experiments/common:/common:ro,z" \
+        --volume "${AGLAIS_CODE:?}/experiments/openstack:/openstack:ro,z" \
+        --volume "${AGLAIS_CODE:?}/experiments/hadoop-yarn:/hadoop-yarn:ro,z" \
+        atolmis/ansible-client:latest \
+        bash
+
+
+
+
+# -----------------------------------------------------
+# Delete everything.
+#[root@ansibler]
+
+    /openstack/bin/delete-all.sh \
+        "${cloudname:?}"
+
+    >   ....
+    >   ....
+
+
+# -----------------------------------------------------
+# Default locations for config and status.
+#[root@ansibler]
+
+    configyml=/tmp/aglais-config.yml
+    statusyml=/tmp/aglais-status.yml
+
+
+# -----------------------------------------------------
+# Create our Aglais configuration.
+#[root@ansibler]
+
+cat > "${configyml:?}" << EOF
+aglais:
+    version: 1.0
+    spec:
+        openstack:
+            cloudname: ${cloudname:?}
+EOF
+
+
+# -----------------------------------------------------
+# Create everything.
+#[root@ansibler]
+
+    /hadoop-yarn/bin/create-all.sh
+
+    >   ....
+    >   ....
+
+PLAY RECAP **************************************************************************************************************************************************************************************************
+gateway                    : ok=4    changed=3    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
+master01                   : ok=4    changed=3    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
+master02                   : ok=4    changed=3    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
+worker01                   : ok=4    changed=3    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
+worker02                   : ok=4    changed=3    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
+worker03                   : ok=4    changed=3    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
+worker04                   : ok=4    changed=3    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
+worker05                   : ok=4    changed=3    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
+worker06                   : ok=4    changed=3    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
+zeppelin      
+
+
+
+# ------------------------------------------------------------------------------------------------------	
+# Open prototype Zeppelin service and copy the Forrest Classifier Notebook & the Bulk loading notebooks
+
+http://zeppelin.aglais.uk:8080/#/notebook/2FRPC4BFS
+http://zeppelin.aglais.uk:8080/#/notebook/2FV6XCUYT
+
+# Run both notebooks
+# Forrest classifier completes after about several hours (aprox. 4) (After modifying the numTrees = 5000)
+# Bulk loading notebooks completes after about several hours (aprox. 5)
```

Comment:
Do we know how long we would expect these to take ?
Is this normal, fatser or slower than expected ?